src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -442,6 +458,52 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
+    /// <summary>
+    /// WorkspaceWriteConverter is used for serializing Workspaces to JSON.
```

Comment:
comment is incorrect.

Reply:
Copy/paste error. The comment has been updated to be correct.